README.md

```
@@ -1,23 +1,41 @@
-- 🔍 Built-in health checking
-- 🛡️ Configurable CORS
-- 🔀 Smart provider-specific request routing
-- 📊 Efficient request/response proxying
-- 💪 Production-ready and battle-tested
-
-## Quick Start
+[![Documentation](https://docs.rs/magicapi-ai-gateway/badge.svg)](https://docs.rs/magicapi-ai-gateway)
+[![License: MIT/Apache-2.0](https://img.shields.io/badge/license-MIT%2FApache--2.0-blue.svg)](LICENSE-MIT)
+[![Docker Pulls](https://img.shields.io/docker/pulls/magicapi1/magicapi-ai-gateway)](https://hub.docker.com/r/magicapi1/magicapi-ai-gateway)
+
+[Quick Start](#quick-start) • 
```

Markdownlint

Link fragments should be valid

(MD051, link-fragments)